docs/content/dagster-cloud/developing-testing/dagster-cloud-yaml.mdx

```diff
@@ -0,0 +1,169 @@
+---
+title: Confiuguration options for dagster_cloud.yaml | Dagster Docs
+description: How to use dagster_cloud.yaml to configure Dagster Cloud
+---
+
+# Dagster_cloud.yaml Guide
+
+<Note>This guide is applicable to Dagster Cloud.</Note>
+
+This dagster_cloud.yaml file is used to configure a Dagster Cloud instance. The file is required to tell Dagster where the code sources are to load into Dagster Cloud but has other options as well. For Hybrid Deployments, secrets are configured in this file as well.
+
+## Where to put the dagster_cloud.yaml file
+
+The dagster_cloud.yaml file should be placed in your dagster project's repository root. Below is an example of a file structure from the [Dagster cloud ETL quickstart](https://github.com/dagster-io/dagster/tree/master/examples/quickstart_etl).
+
+```shell
+quickstart_etl
+├── README.md
+├── quickstart_etl
+│   ├── __init__.py
+│   ├── assets
+├── quickstart_etl_tests
+├── dagster_cloud.yaml
+├── pyproject.toml
+├── setup.cfg
+└── setup.py
+```
+
+## Code Source
+
+The only **mandatory** section in the dagster_cloud.yaml file is setting up your code locations. If you have multiple code locations, you will have multiple code locations within a single dagster_cloud.yaml file. Dagster code can be sourced from a package, a python file, or a module.
```

Comment:
This section is called Code Source but part of it is talking about the required "locations" list I think? It's both mandatory that you have a list of locations in your dagster_cloud.yaml, and then each of those locations in the locations list needs to have a code_source and a location_name key